tests/unit/test_logging.py

```diff
@@ -0,0 +1,42 @@
+import logging
+import os
+import time
+
+from pip._internal.utils.logging import IndentingFormatter
+
+
+class Test_IndentingFormatter(object):
```

Comment:
I would call this `TestIndentingFormatter` as that is the predominant style in the code base and you don't lose anything by removing the underscore.